.github/pull_request_template.md

```diff
@@ -0,0 +1,46 @@
+## Description
+
+✅ ...
+✅ ...
+
+## References
+
+[Trello-XXXX](https://bcg-platinion.atlassian.net/browse/ER-XXXX)
+
+## Types of changes
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Refactor (non-breaking change which improves existing functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to change)
+
+<!--
+  ⚠️ IMPORTANT ⚠️
+  - DELETE EVERYTHING FROM THIS COMMENT ONWARDS BEFORE MERGING PR
+  - Squash merge with the following message:
+    <type>(<epicCode>): [<ticketCode>] (scope) - shortText
+    i.e.: feat(er-324): [er-2563] (AvtaleGiro) - Agreement summary screen
```

Comment:
Since we don't have any epicCode or ticketCode then this example should be changed or removed if we are keeping the template

Reply:
Removed it